src/annotator/integrations/vitalsource.ts

```diff
@@ -337,7 +337,7 @@ export class VitalSourceContentIntegration
     // currently limit page number selectors to PDFs until more is understood
     // about when EPUB page numbers are reliable/likely to remain stable.
     const bookInfo = this._bookElement.getBookInfo();
-    if (bookInfo.format === 'pbk') {
+    if (bookInfo.format === 'pbk' && typeof pageIndex === 'number') {
```

Comment:
I believe this check is redundant. It would not be a critical problem in PDF books if the page number was missing, since the synthetic CFI should still be present.

Reply:
I wonder if we could bake that context into source (i.e. comment as to why `pageIndex` is part of this conditional)? Or do you feel that it is self-documenting?

Reply:
I'll add a note.

Reply:
Hmm, I think the comment above is probably adequate. I did push a commit to update the docs for the `PageInfo.index` property which the `pageIndex` value comes from.